scripts/zones/Palborough_Mines/mobs/Copper_Beetle.lua

```diff
@@ -22,13 +22,15 @@ function onMobDespawn(mob)
         local checkPH = GetServerVariable("[PH]Bu_Ghi_Howlblade");
         if (ToD <= os.time() and GetMobByID(Bu_Ghi_Howlblade):isDead()) then
             -- Set ToD, DisallowRespawn PH repop, set NM respawn time
-            if (math.random(1,10) == 5 and checkPH == 0) then
```

Comment:
no point in repeating the same commands once the already happened I tested those beetles and the nm last night there is no problem with having the var checked at the same time as the random number, being together doesn't make it "not used"